index.bs

```diff
@@ -615,6 +615,8 @@ information about the platform version on which a given [=user agent=] is execut
 [[!RFC8941]].
 
 [=User agents=] SHOULD return a version with a platform-specific format that allows differentiating significant platform versions by running these steps:
+    1. If the platform is Linux:
+    	1. Return the empty string.
     1. Let |platformVersionComponentList| be a [=list=].
```

Comment:
Unrelated to your commit, but one small suggestion here which might help with readability would be to start this line with "Else, " and perhaps pull "Use per-platform logic" into the sentence. So something like:

`Else, let |platformVersionComponentList| be a [=list=] and use per-platform logic:` 

What do you think @kyraseevers?

Reply:
I agree - I think this will help readability significantly. I will implement and re-commit.